module/applications/sheets/base-actor-sheet.mjs

```diff
@@ -643,6 +644,20 @@ export default class CrucibleBaseActorSheet extends api.HandlebarsApplicationMix
 
   /* -------------------------------------------- */
 
+  #prepareLanguages() {
+    const categories = SYSTEM.ACTOR.LANGUAGE_CATEGORIES;
+    const options = [];
+    for ( const [value, {label, category}] of Object.entries(SYSTEM.ACTOR.LANGUAGES) ) {
+      const currOption = { value, label };
+      if ( category && categories[category] ) currOption.group = categories[category];
+      options.push(currOption);
+    }
+    const groups = options.reduce((acc, {group}) => (group && !acc.includes(group)) ? acc.concat(group) : acc, []);
+    return { options, groups };
```

Comment:
We don't need to pass the `groups` parameter unless we need manual control over the order in which groups are listed
```suggestion
    return options;
```